jquery.simulate.js

```diff
@@ -200,8 +200,8 @@ $.extend( $.simulate.prototype, {
 		return event;
 	},
 
-	dispatchEvent: function( elem, type, event ) {
-		if ( elem[ type ] ) {
+	dispatchEvent: function( elem, type, event, options ) {
+		if ( elem[ type ] && !options ) {
```

Comment:
This is an extremely confusing condition, since `options` are meaningless at this point. Do you know why this condition event exists instead of just always going through `dispatchEvent()`?